Sources/HealthKitOnFHIR/Resources/HKSampleMapping.json

```
@@ -1,4 +1,93 @@
 {
+    "HKElectrocardiogram": {
+      "categories": [
+        {
+          "code": "vital-signs",
+          "display": "Vital Signs",
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category"
```

The category code for an ECG should be "procedure", as noted in the FHIR documentation [here](https://www.hl7.org/fhir/codesystem-observation-category.html).

```suggestion
          "code": "procedure",
          "display": "Procedure",
          "system": "http://terminology.hl7.org/CodeSystem/observation-category"
```